### PR TITLE
Vehicle raycast implementation

### PR DESCRIPTION
**Pull request by richtrr**
Interfaces are added to support the Bullet RayCast Vehicle and demo "examples/webgl_demo_vehicle/"

### CODE REVIEW

File: ammo.idl [hunk 1]
```diff
@@ -576,6 +576,9 @@ interface btDiscreteDynamicsWorld {
   void removeConstraint(btTypedConstraint constraint);
 
   long stepSimulation(float timeStep, optional long maxSubSteps, optional float fixedTimeStep);
+  void addVehicle (btActionInterface vehicle);
+  void removeVehicle (btActionInterface vehicle);
```
Comment (willeastcott) [inline ammo.idl]: These functions are obsolete. You're supposed to use add/removeAction instead:

http://bulletphysics.org/Bullet/BulletFull/classbtDynamicsWorld.html#ab897455739081304249e7af787cafa19

**Comment by kripken** [comment]
Looks good, thanks!

Just instead of adding temp.js, just move that to ammo.js (the build system emits temp.js for testing convenience, but in the repo we should only have ammo.js).

File: examples/webgl_demo_vehicle/index.html [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+            }
+
+        </style>
+    </head>
+    <body>
+        <div id="container"><br /><br /><br /><br /><br />Loading...</div>
+		<div id="speedometer">0.0 km/h</div>
+        <div id="info">Ammo.js Raycast vehicle demo<br>Press W,A,S,D to move.</div>
+
+		<script src="../../builds/temp.js"></script>
```
Comment (kripken) [inline examples/webgl_demo_vehicle/index.html]: for the demo this should use `ammo.js`

**Comment by kripken** [comment]
Great, thanks! :)

And thanks @willeastcott for reviewing the API parts.

**Comment by richtrr** [comment]
You  thanks :) Ammo and Emscripten very cool!

**Pull request merged**